sdk/storage/azure-storage-blob/azure/storage/blob/_blob_service_client.py

```diff
@@ -462,6 +462,13 @@ def find_blobs_by_tags(self, filter_expression, **kwargs):
             To specify a container, eg. "@container='containerName' and \"Name\"='C'"
         :keyword int results_per_page:
             The max result per page when paginating.
+        :keyword list[str] include:
+            Specifies one or more additional datasets to include in the response.
+            Options include: 'versions'.
+
+            .. versionadded:: 12.13.0
+                This parameter was introduced in API version '2021-08-06'.
```

Comment:
Is this accurate for STG84? Can change this to reflect the correct API version (since I think the version would be bumped up for this release? but just don't know the value)

Reply:
The STG84 version should be 12.14.0 barring any unforeseen patch releases we end up needing.